flaskbb/plugins/__init__.py

```diff
@@ -32,15 +121,24 @@ def installable(self):
     def uninstallable(self):
         """Is ``True`` if the Plugin can be uninstalled."""
         if self.installable:
-            group = SettingsGroup.query.\
-                filter_by(key=self.settings_key).\
+            group = SettingsGroup.query. \
+                filter_by(key=self.settings_key). \
                 first()
             if group and len(group.settings.all()) > 0:
                 return True
             return False
         return False
 
+    def this_version_installed(self):
+        if self.uninstallable():
+            if Setting.as_dict(self.settings_key).get('version',None)==self.version:
```

Comment:
This indicates that we expect to store the current version in the database as well? Wouldn't it better if we compare the installed alembic version? And it looks like that this isn't used anywhere.